code/game/area/Space_Station_13_areas.dm

```diff
@@ -1036,6 +1039,8 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "Prison Wing"
 	icon_state = "sec_prison"
 	mood_bonus = -4
+	mood_job_allowed = list(JOB_NAME_HEADOFSECURITY,JOB_NAME_WARDEN)
```

Comment:
HoS and Warden hate being in the prison wing (where they have access?)
They are "trapped"?

Reply:
nevermind, just noticed it's reverse.

Why do sec offs hate it there as well? They should be included in the exemption